src/pages/mobile/NameDetails.vue

```diff
@@ -55,15 +63,32 @@ export default {
       pointers: DetailsNamePointers,
     },
   }),
-  computed: mapState('names', {
-    details({ owned }) {
-      return owned && owned.names.find(({ name }) => name === this.name);
-    },
-  }),
+  computed: {
+    ...mapGetters(['currentNetwork']),
+    ...mapState('names', {
+      details({ owned }) {
+        return owned && owned.names.find(({ name }) => name === this.name);
+      },
+    }),
+  },
   async mounted() {
     const id = setInterval(() => this.$store.dispatch('names/fetchOwned'), 3000);
+    const address = this.details.pointers && this.details.pointers[0].id;
```

Comment:
use `getAddressByNameEntry` from `lib/utils` because it makes check of `key` field (it should by `account_pubkey`), otherwise, this code may work wrong if a name will point to several entities